modules/serverless-invoking-go-functions-cloudevent.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-go-functions.adoc
+
+[id="serverless-invoking-go-functions-cloudevent_{context}"]
+= Function triggered by CloudEvent
```

Comment:
I like this section a lot better without the numbering!

The only suggestion I have is for the title. It might sound better if Functions was plural, and either add "a/an" in front of CloudEvent/HTTP request, or made those plural as well. 

Like "Functions triggered by a CloudEvent" ?